Runtimes/Core/cmake/caches/Vendors/Apple/arm64e-MacOSX.cmake

```diff
@@ -9,8 +9,6 @@ set(CMAKE_Swift_COMPILER_TARGET "arm64e-apple-macosx${CMAKE_OSX_DEPLOYMENT_TARGE
 set(SwiftCore_ARCH_SUBDIR arm64e CACHE STRING "")
 set(SwiftCore_PLATFORM_SUBDIR macosx CACHE STRING "")
 
-list(APPEND CMAKE_C_FLAGS "-darwin-target-variant" "arm64e-apple-ios13.1-macabi")
-list(APPEND CMAKE_CXX_FLAGS "-darwin-target-variant" "arm64e-apple-ios13.1-macabi")
-list(APPEND CMAKE_Swift_FLAGS "-target-variant" "arm64e-apple-ios13.1-macabi")
+set(SwiftCore_TARGET_VARIANT_COMPILER_TARGET "arm64e-apple-ios13.1-macabi" CACHE STRING "")
```

Comment:
Should we allow the user to configure the deployment target (like we do for the settings above)?

Reply:
Yeah, that's reasonable.